draft/spec/index.html

```diff
@@ -854,7 +854,17 @@ <h2>Logs Directory</h2>
                 wrong, while others may wish to only store a log entry if an intervention was required.
             </p>
         </blockquote>
+    </section>
 
+    <section id="extensions">
+        <h2>Extensions directory</h2>
+        <p>
+            The base directory of an OCFL Object MAY contain a directory named <code>extensions</code> for the purposes
+            of extending the functionality of an OCFL Object. Sub-directories of the extensions directory MUST be named
+            according to a registered extension name. OCFL Validators MUST ignore any content in extensions that are
+            not understood. The specific structure and function of the extension, as well as a declaration of the
+            registered extension name MUST be defined in the OCFL Extensions Registry.
```

Comment:
I'd go for SHOULD instead of MUST again, I'd also put any such language next to the sentence about the name

Reply:
SHOULD would leave open the possibility of registered and unregistered namespace collisions -- two extensions with the same name but different functions. By making it a MUST we reduce this possibility.